tests/subsys/storage/flash_map/prj_native_posix.conf

```diff
@@ -3,3 +3,4 @@ CONFIG_FLASH=y
 CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
 CONFIG_FLASH_AREA_CHECK_INTEGRITY=y
+CONFIG_ZTEST_NEW_API=y
```

Comment:
it seems that the prj_native_posix.conf, prj_native_posix_64.conf, they are the same with prj.conf, no any special configs related to platform, so you could delete them.